src/map/battle.c

```diff
@@ -2064,7 +2064,7 @@ static int battle_calc_skillratio(int attack_type, struct block_list *src, struc
 					break;
 				}
 				case AM_DEMONSTRATION:
-					skillratio += 20 * skill_lv;
+					skillratio += skill_lv * 1;
```

Comment:
It's same as just skill_lv without any multiplication.

Also, the source you quote is not reliable neither states this change.
If anyone can confirm it would be better